wenv

```diff
@@ -324,6 +324,7 @@ OPTIONS
     [[ -z "$1" ]] && return 1
     local wenv="$1"
 
+    mkdir $WENV_CFG/wenvs/$(dirname $wenv)
```

Comment:
We'll need to use the `-p` flag in case there are nested directories (to create all parent directories as well).